pkg/cmd/get.go

```diff
@@ -212,6 +212,7 @@ func getShoot(name string) {
 		}
 	}
 	Client, err = clientToTarget("garden")
+	gardenName := target.Stack()[0].Name
```

Comment:
You can move this right after the target read operation, do not put it between another err check.